exchange/exchange_test.go

```diff
@@ -4073,6 +4074,132 @@ func TestAuctionDebugEnabled(t *testing.T) {
 
 }
 
+func TestExperimentConfigs(t *testing.T) {
+	//This test just pass experiment ads cert configs to HoldAuction
```

Comment:
Consider choosing a more descriptive test name and removing the comment. Looks like the test assets and error, but is that a real error from trying to determine if test.com has a call sign registered?

Reply:
Renamed to `TestPassExperimentConfigsToHoldAuction`. This test runs HoldAuction with MockSigner and only tests correct data was passed to the signer.